### PR TITLE
[MIG][10.0] mrp analytic

### PR DESCRIPTION
**Pull request by AaronHForgeFlow**
No extra features added or removed from v8

### CODE REVIEW

**Comment by LoisRForgeFlow** [comment]
Can you please check travis? some tests are failing.

**Comment by AaronHForgeFlow** [comment]
@lreficent this is super green now :rocket:

File: mrp_analytic/views/mrp_view.xml [hunk 1]
```diff
@@ -2,18 +2,6 @@
 <openerp>
     <data>
 
-        <record id="mrp_production_tree_view" model="ir.ui.view">
-            <field name="name">mrp.production.tree</field>
-            <field name="model">mrp.production</field>
-            <field name="inherit_id" ref="mrp.mrp_production_tree_view"/>
-            <field name="arch" type="xml">
-                <xpath expr='//field[@name="product_uom"]' position="after">
```
Comment (LoisRForgeFlow) [inline mrp_analytic/views/mrp_view.xml]: why are you removing this file? you can port it easily: `product_uom` -> `product_uom_id`

File: mrp_analytic/views/mrp_view.xml [hunk 2]
```diff
@@ -2,18 +2,6 @@
 <openerp>
     <data>
```
Comment (LoisRForgeFlow) [inline mrp_analytic/views/mrp_view.xml]: openerp and data out odoo in :smile:

**Review by LoisRForgeFlow — COMMENTED** [review]
Some little comments

**Comment by AaronHForgeFlow** [comment]
@lreficent Done!

**Review by LoisRForgeFlow — APPROVED** [review]
LGTM now :+1:

File: README.md [hunk 3]
```diff
@@ -32,6 +32,7 @@ addon | version | summary
 [procurement_analytic](procurement_analytic/) | 10.0.1.0.0 | This module adds analytic account to procurements
 [product_analytic](product_analytic/) | 10.0.1.0.0 | Add analytic account on products and product categories
 [purchase_procurement_analytic](purchase_procurement_analytic/) | 10.0.1.0.0 | This module sets analytic account in purchase order line from procurement analytic account
+[sale_analytic_distribution](sale_analytic_distribution/) | 10.0.1.0.0 | Sales Analytic Distribution
```
Comment (rousseldenis) [inline README.md]: Not concerned by this module

File: mrp_analytic/models/mrp_production.py [hunk 4]
```diff
@@ -0,0 +1,12 @@
+# -*- coding: utf-8 -*-
+# © 2015 Eficent - Jordi Ballester Alomar
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+from openerp import models, fields
```
Comment (rousseldenis) [inline mrp_analytic/models/mrp_production.py]: odoo

File: mrp_analytic/tests/test_mrp_analytic.py [hunk 5]
```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
+# © 2015 Pedro M. Baeza - Antiun Ingeniería
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+from openerp.tests import common
```
Comment (rousseldenis) [inline mrp_analytic/tests/test_mrp_analytic.py]: odoo

File: setup/sale_analytic_distribution/odoo/__init__.py [hunk 6]
```diff
@@ -0,0 +1 @@
+__import__('pkg_resources').declare_namespace(__name__)
```
Comment (rousseldenis) [inline setup/sale_analytic_distribution/odoo/__init__.py]: All setup related stuff is wrong (wrong module)

**Review by rousseldenis — CHANGES_REQUESTED** [review]
Code review
Minor comments

**Comment by rousseldenis** [comment]
@aheficent This case, I wiil create a mrp_procurement_analytic that inherits from this

**Comment by rousseldenis** [comment]
@aheficent Opened https://github.com/Eficent/account-analytic/pull/1 to fix this.

This needs also a rebase

**Review by rousseldenis — APPROVED** [review]

**Review by MiquelRForgeFlow — APPROVED** [review]

**Comment by AaronHForgeFlow** [comment]
Done some squashing

**Comment by pedrobaeza** [comment]
Do you think the step from openerp to odoo word deserves an isolated commit?

**Comment by AaronHForgeFlow** [comment]
yes, better to squash it. Done.

**Pull request merged**